src/path_functions.cpp

```diff
@@ -140,13 +140,17 @@ QString AOApplication::get_pos_path(const QString& pos, const bool desk)
   QString desk_override = read_design_ini("overlays/" + f_background, get_background_path("design.ini"));
```

Comment:
design_ini override is checked here. But overlays/ folder override is only checked *after* the image is attempted to be loaded, inside the background layer itself? Doesn't that order betray the PR description?